android/modules/android/src/java/ti/modules/titanium/android/notificationmanager/NotificationManagerModule.java

```diff
@@ -36,6 +37,11 @@
 	protected static final int PENDING_INTENT_FOR_BROADCAST = 2;
 	protected static final int PENDING_INTENT_MAX_VALUE = PENDING_INTENT_FOR_SERVICE;
 
+	private static NotificationManager notificationManager = null;
+	private static NotificationChannel defaultChannel = null;
+
+	public static final String DEFAULT_CHANNEL_ID = "defaultChannel";
```

Comment:
Do you think maybe we should use a prefixing notation to help avoid name collision?
Something like: `"ti_default_channel"`
(Kind of like how it's done with resources.)